etcd/TAG

```diff
@@ -1 +1 @@
-3.5.4.1
```

Comment:
I didn't think it was necessary to update the tag if it was just a README modification.

Reply:
The etcd image is a ubuntu-based image. So I intent to update the base image.